js/styles/inlineStyle.js

```diff
@@ -11,6 +11,24 @@ module.exports = {
     }
   },
 
+  iPadVerticalVideoStyle: {
```

Comment:
I am not sure that using the changing class name as a way to apply a different style for the main video element is completely safe. 
@chibzu, what do you think?